fvtest/tril/tril/CMakeLists.txt

```diff
@@ -23,44 +23,88 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake/modules"
 	${CMAKE_MODULE_PATH})
 
 include(OmrUtility)
-include(FindYACC)
-include(FindLEX)
-
-find_package(YACC REQUIRED)
-find_package(LEX REQUIRED)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-list(APPEND TRIL_FLEX_FLAGS)
+find_package(YACC)
+find_package(LEX)
+
+if (YACC AND LEX)
+
+	include(FindYACC)
+	include(FindLEX)
+
+	list(APPEND TRIL_FLEX_FLAGS)
+
+	# There is no unistd.h on Windows, so if win_flex.exe is in use,
+	# the --wincompat option must be added.
+	if(OMR_HOST_OS STREQUAL "win")
+		list(APPEND TRIL_FLEX_FLAGS
+			"--wincompat"
+		)
+	endif()
+
+	omr_stringify(TRIL_FLEX_OPTIONS ${TRIL_FLEX_FLAGS})
+	YACC_TARGET(tril_parser tril.y ${CMAKE_CURRENT_BINARY_DIR}/tril.parser.c
+		DEFINES_FILE ${CMAKE_CURRENT_BINARY_DIR}/tril.parser.h)
+	LEX_TARGET(tril_scanner tril.l ${CMAKE_CURRENT_BINARY_DIR}/tril.scanner.c
+		COMPILE_FLAGS ${TRIL_FLEX_OPTIONS}
```

Comment:
I know it's outside the scope of this PR, but COMPILE_FLAGS should have been a multivalue keyword, so the explicit flag list stringification on L49 wouldn't have to be done.

Reply:
Please open an issue so we don't forget about it.

Reply:
After thinking a bit more about this, I've decided not to make this change. The bison equivalent can only take a single string argument, and the two interfaces should match.